docs/dynamic-links/usage/index.md

```diff
@@ -36,6 +36,53 @@ a Dynamic Link on iOS or Android, they can be taken directly to the linked conte
+
+![firebase console dynamic link first step](https://images.prismic.io/invertase/4152f98c-b4e9-4561-a790-a0750a0392bb_Screenshot+2020-05-07+at+09.26.47.png?auto=compress,format)
+
+2. Create a dynamic link with your domain in the Firebase console. The freshly created dynamic link URL will appear in the table.
+
+![firebase console dynamic link second step](https://images.prismic.io/invertase/7e5c7a61-899f-45e5-ab4e-ddf3cfee636a_Screenshot+2020-05-07+at+10.26.22.png?auto=compress,format)
+
+## iOS Setup
+
+To setup Dynamic Links on iOS, it is a **prerequisite** that you have an Apple developer account [setup](https://developer.apple.com/programs/enroll/).
+
+1. Add an `App Store ID` & `Team ID` to your app in your Firebase console. If you do not have an `App Store ID` yet, you can put any number in here for now. Your `Team ID` can be found in your Apple developer console.
+
+![ios dynamic link first step](https://images.prismic.io/invertase/30e302f6-a7bb-4e37-98fd-7115dac4c1f1_Screenshot+2020-05-07+at+09.10.02.png?auto=compress,format)
+
+2. Test the domain you created in your Firebase console (first step in `Firebase Setup`). Go to the following location in your browser `[your domain]/apple-app-site-association`. The response will have a `details` array property containing an object that has the property `appID`. That will be your app's app ID (It may take some
+   time for your domain to register). Please ensure it is registered before proceeding.
+
+3. Once you're sure your domain is registered, you need to head over to your Apple developer console and create a provisioning profile for your app. Please ensure you've enabled the `Associated Domain` capability which you should check before proceeding.
+
+![ios dynamic link third step](https://images.prismic.io/invertase/12d6e692-c2b1-47f5-98aa-f364c928114a_Screenshot+2020-05-07+at+10.03.45.png?auto=compress,format)
+
+4. Open your project in Xcode and open your app under the `TARGETS` header. Click the `Signing & Capabilities` tab. You will need to ensure your `Team` is registered, and your `Provisioning Profile` field is completed. Please add the domain you created in your
+   Firebase console to the `Associated Domains` and prefix with `applinks:`
+
+![ios dynamic link fourth step](https://images.prismic.io/invertase/d7a8696f-bef1-44a4-b048-aab7bb534f71_Screenshot+2020-05-07+at+09.55.53.png?auto=compress,format)
+
+5. Click the `Info` tab, and add a `URL Type` to your project. The `Identifier` can be called `Bundle Id` or whatever you wish. Add your bundle identifier to the `URL Schemes` property.
+
+![ios dynamic link fifth step](https://images.prismic.io/invertase/cb029ba6-ad40-494e-a3f6-2aacaff494d1_Screenshot+2020-05-07+at+10.16.16.png?auto=compress,format)
+
+6. To test the dynamic link works, you will need to use a real device as it will not work on a simulator. The notes app is a good place to paste you link and test it opens your app (ensure the app is installed on the device).
```

Comment:
Is this to say FB dynamic links don't work on simulators? Is that particular to RNFB or to the Firebase SDK in general?

Reply:
I think it is possible to test it on the simulator, @russellwheatley you may have accidentally disabled it on the simulator, see: https://youtu.be/KLBjAg6HvG0?t=500

Reply:
Those points on those video might be good to add to troubleshooting links not opening on iOS section? And also documenting how to test the installed & non-installed flows on iOS - see https://stackoverflow.com/questions/39605657/how-can-i-test-firebase-dynamic-links-if-my-app-is-not-in-the-app-store that summarizes it well.

Reply:
If you listen from 7 minutes on, Todd says that Dynamic Links do not work on the simulator: https://www.youtube.com/watch?v=KLBjAg6HvG0&feature=youtu.be&t=500. The documentation is based on iOS screencast pt.1. I can add documentation on how to test without the app being installed. I'll also watch iOS screencast pt.2 to see if there's anything else that could be added to the documentation.

Reply:
@russellwheatley fair enough re the simulator - be good to document those troubleshooting steps still I think